src/components/game/game.jsx

```diff
@@ -118,5 +120,3 @@ Game.propTypes = {
 };
 
 
-export default Game;
```

Comment:
It's good to keep your components named and export the components this way since it creates a named component in React Developer Tools for Google Chrome.